src/test/unit/profile-data.test.js

```diff
@@ -187,40 +187,49 @@ describe('process-profile', function() {
 
       expect(profile.threads[0].samples.time[0]).toEqual(0);
       expect(profile.threads[0].samples.time[1]).toEqual(1);
+
+      // 1 second later than the same samples in the main process because the
+      // content process' start time is 1s later.
       expect(profile.threads[2].samples.time[0]).toEqual(1000);
       expect(profile.threads[2].samples.time[1]).toEqual(1001);
-      expect(profile.threads[0].markers.time[0]).toEqual(0);
+
+      // Now about markers
+      expect(profile.threads[0].markers.time[0]).toEqual(1);
       expect(profile.threads[0].markers.time[1]).toEqual(2);
-      expect(profile.threads[0].markers.time[2]).toEqual(4);
-      expect(profile.threads[0].markers.time[3]).toEqual(5);
+      expect(profile.threads[0].markers.time[2]).toEqual(3);
+      expect(profile.threads[0].markers.time[3]).toEqual(4);
+      expect(profile.threads[0].markers.time[4]).toEqual(5);
```

Comment:
I had to change the both the indices and the values because I changed the test data